src/App/Fossa/Config/Container/Analyze.hs

```diff
@@ -95,6 +98,7 @@ cliParser =
       )
     <*> metadataOpts
     <*> imageTextArg
+    <*> switch (long "experimental-scanner" <> short 'x' <> help "Uses experimental fossa native container scanner")
```

Comment:
```suggestion
    <*> switch (long "experimental-scanner" <> help "Uses experimental fossa native container scanner")
```
Explicit goals for the `experimental-` prefix were:
- Make it extremely clear to users that they're doing something outside our normal SLA.
- Avoid colliding with another flag (especially since we maintain flags ~forever).

Would prefer we do not provide a shorthand version.

Reply:
Agree with your assessment; I will remove the `x` flag.